docker-distribution/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:3.5-jre8-alpine
+MAINTAINER varkockova.a@gmail.com
+
+USER root
+COPY build.gradle /
+WORKDIR /
+ARG PLUGIN_VERSION
+# ARGs are not persisted so we must store value of ARG to ENV
+ENV PLUGIN_VERSION=$PLUGIN_VERSION
```

Comment:
This is how it works :) The problem is that ARG value is not persisted in the image and so we must store it to ENV.